src/utils/nested-form.ts

```diff
@@ -63,12 +63,20 @@ export function nestedForm(
     ...form,
     path(field?: string | number) {
       const fullPath = path && field ? `${path}.${field}` : path ? path : field
-      if ("path" in form) return form.path(path as any)
+      
+      if ("path" in form) {
+        return form.path(path as any)
+      }
+
```

Comment:
Praise: This seems like a very cool util in the way that you utilize it for products and the nested forms